src/drivers/gps/gps.cpp

```diff
@@ -510,6 +510,8 @@ int GPS::setBaudrate(unsigned baud)
 	int speed;
 
 	switch (baud) {
+	case 4800:   speed = B4800;   break;
```

Comment:
Now that you're saying it - it will slow down scanning. Can you please find out and remove the other one?